homework2/main.py

```diff
@@ -0,0 +1,28 @@
+import string
+import random
+
+
+from flask import Flask
+
+import pandas as pd
+
+
+app = Flask(__name__)
+
+@app.route("/generate_password")
+def hello_world():
+    generate_pass = ''.join([random.choice(
+                        string.ascii_lowercase + string.ascii_uppercase + string.digits + string.punctuation)
+                        for n in range(random.randint(10, 20))])
```

Comment:
Try to use choices not choice

Reply:
sure - let me check differences between

Reply:
@mikolaz27 
added file requirements.txt changed .gitignore
added choices realization